index.bs

```diff
@@ -839,6 +838,20 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
                     bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
 
+                1.  Let |attestationPresentationPreference| be the value of |options|.{{MakePublicKeyCredentialOptions/attestation}}.
```

Comment:
IIUC, we ought to rather add `attestationPresentationPreference` as an item of the `credentialCreationData` struct,  created in a prior step, and then utilize `credentialCreationData.attestationPresentationPreference`  in this step.